### PR TITLE
for review - issue 26

### PR DESCRIPTION
**Pull request by TheRealVish**
1. initialise anarchy.db 
2. function to insert logs in to db
3. added initial logs in routes

If this approach is acceptable I will add telemetric commits in all relevant locations.

### CODE REVIEW

File: src/llm_vm/utils/db_initializer.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import sqlite3
+import time
+
+def initialize_db():
+    print('Initialising database...')
```
Comment (cartazio) [inline src/llm_vm/utils/db_initializer.py:5]: the print should be to standard error

**Comment by cartazio** [comment]
so i'm looking at this, and i'm thinking "wait" we dont actually want the LOGGGINg to be persisted, we want the information about models we've fine tuned to be persisted. i'm not sure what the best practices are for various logging htings in python, https://docs.python.org/3/howto/logging.html to std error might be a way to do it.

File: src/llm_vm/utils/logger.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import sqlite3
+import time
+
+def log_telemetry(logSource, logLevel, logMessage, logData, logType):
```
Comment (cartazio) [inline src/llm_vm/utils/logger.py:4]: this module is dead code now right?

File: src/llm_vm/utils/db_initializer.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import sqlite3
+import time
+import sys
+from llm_vm.utils.stdlog import setup_logger
+
+def initialize_db():
```
Comment (cartazio) [inline src/llm_vm/utils/db_initializer.py:6]: module is also dead code now?

**Comment by mmirman** [comment]
This also addresses #71

**Comment by subramanya1997** [comment]
I think it would be nice if you could offload logs to multiple sources like Kafka, file, and db.

**Pull request closed without merging**